dvc/ignore.py

```diff
@@ -52,16 +52,12 @@ def __init__(self, pattern_list, dirname):
         ]
 
     @classmethod
-    def from_files(cls, ignore_file_path, fs, root):
-        assert os.path.isabs(ignore_file_path)
-        dirname = os.path.normpath(os.path.dirname(ignore_file_path))
-        ignore_file_rel_path = os.path.relpath(ignore_file_path, root)
```

Comment:
This was really just a name for information purposes, so we could just pass it down here instead of using `fs.fs_root` or `root`, which previously gave an impression of being used here for some functional purposes.